hack/update-mocks.sh

```diff
@@ -26,16 +26,6 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 # Explicitly opt into go modules, even though we're inside a GOPATH directory
 export GO111MODULE=on
 
-_tmp="${KUBE_ROOT}/_tmp_build_tag_files"
-mkdir -p "${_tmp}"
-
-function cleanup {
-    rm -rf "$_tmp"
-    rm -f "tempfile"
```

Comment:
we setup an actual tempfile below, instead of using a file named "tempfile" in pwd, which is not gitignored.